client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/packages-step/move-item.js

```diff
@@ -104,11 +103,22 @@ const MoveItemDialog = ( props ) => {
 
 	const onClose = () => props.closeItemMove( orderId, siteId );
 
+	const buttons = [
+		{ label: translate( 'Cancel' ) },
```

Comment:
Do you think it's worth it to add `onClick: onClose` to this button (and the other `Close` buttons in the dialogs), just to be more explicit?